README.md

```diff
@@ -1,5 +1,15 @@
 # Documentation
 
+* [Getting Started](#getting-started)
```

Comment:
ooi, do we have a 'standard' for where to put the toc? I believe I have posted some docs where the top lives above the first `#` top level heading for instance :-)

Reply:
We currently say ["at the top"](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#general-requirements) but that can be before or after the optional top-level link which refers to the doc itself.